<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkoutTimerClass</class>
 <widget class="QMainWindow" name="WorkoutTimerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkoutTimer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WorkoutTimer.ico</normaloff>WorkoutTimer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QGroupBox" name="routineGroupBox">
      <property name="title">
       <string>Routine</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="routineListView">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QToolButton" name="addToolButton">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="workouttimer.qrc">
              <normaloff>:/WorkoutTimer/Resources/add.png</normaloff>:/WorkoutTimer/Resources/add.png</iconset>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QToolButton" name="removeToolButton">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset resource="workouttimer.qrc">
              <normaloff>:/WorkoutTimer/Resources/delete.png</normaloff>:/WorkoutTimer/Resources/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="upToolButton">
            <property name="text">
             <string>Up</string>
            </property>
            <property name="icon">
             <iconset resource="workouttimer.qrc">
              <normaloff>:/WorkoutTimer/Resources/arrow_up.png</normaloff>:/WorkoutTimer/Resources/arrow_up.png</iconset>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="downToolButton">
            <property name="text">
             <string>Down</string>
            </property>
            <property name="icon">
             <iconset resource="workouttimer.qrc">
              <normaloff>:/WorkoutTimer/Resources/arrow_down.png</normaloff>:/WorkoutTimer/Resources/arrow_down.png</iconset>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="settingsGroupBox">
         <property name="title">
          <string>Interval Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="intervalNameLabel">
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="buddy">
             <cstring>nameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="intervalTypeLabel">
            <property name="text">
             <string>Interval Type:</string>
            </property>
            <property name="buddy">
             <cstring>intervalTypeComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="intervalTypeComboBox">
            <item>
             <property name="text">
              <string>Ladder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pyramid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plateau</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Constant</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="timeStepLabel">
            <property name="text">
             <string>Time Step:</string>
            </property>
            <property name="buddy">
             <cstring>timeStepSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="timeStepSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="startingMultiplierLabel">
            <property name="text">
             <string>Starting Multiplier:</string>
            </property>
            <property name="buddy">
             <cstring>startingMultiplierSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="startingMultiplierSpinBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="endingMultiplierLabel">
            <property name="text">
             <string>Ending Multiplier:</string>
            </property>
            <property name="buddy">
             <cstring>endingMultiplierSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="endingMultiplierSpinBox"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="timerStyleLabel">
            <property name="text">
             <string>Timer Style:</string>
            </property>
            <property name="buddy">
             <cstring>timerStyleComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="timerStyleComboBox">
            <item>
             <property name="text">
              <string>Countdown</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Count up</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="peakMultiplierLabel">
            <property name="text">
             <string>Peak Multiplier:</string>
            </property>
            <property name="buddy">
             <cstring>peakMultiplierSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="peakMultiplierSpinBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="stepsAtPeakLabel">
            <property name="text">
             <string>Steps at Peak:</string>
            </property>
            <property name="buddy">
             <cstring>stepsAtPeakSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="stepsAtPeakSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start Routine</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>intervalTypeComboBox</tabstop>
  <tabstop>timeStepSpinBox</tabstop>
  <tabstop>startingMultiplierSpinBox</tabstop>
  <tabstop>peakMultiplierSpinBox</tabstop>
  <tabstop>endingMultiplierSpinBox</tabstop>
  <tabstop>stepsAtPeakSpinBox</tabstop>
  <tabstop>timerStyleComboBox</tabstop>
  <tabstop>addToolButton</tabstop>
  <tabstop>removeToolButton</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources>
  <include location="workouttimer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>WorkoutTimerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>WorkoutTimerClass</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>WorkoutTimerClass</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>routineListView</sender>
   <signal>indexesMoved(QModelIndexList)</signal>
   <receiver>WorkoutTimerClass</receiver>
   <slot>routinesMoved(QModelIndexList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load()</slot>
  <slot>save()</slot>
 </slots>
</ui>
